src/js/tracks/text-track.js

```diff
@@ -195,7 +195,8 @@ TextTrack.prototype.constructor = TextTrack;
  * cuechange - One or more cues in the track have become active or stopped being active.
  */
 TextTrack.prototype.allowedEvents_ = {
-  'cuechange': 'cuechange'
+  'cuechange': 'cuechange',
+  'loadeddata': 'loadeddata'
```

Comment:
we actually don't want to add `loadeddata` here. We can still fire it but it should be a "private" event, like `modechange`.